Docs/User_Guides/WebUI_Extension/TTS-SETUP-GUIDE.md

```
@@ -0,0 +1,724 @@
+  # Voice cloning settings
+  enable_voice_clone: true
+  voice_clone_min_duration: 3.0  # seconds
+  voice_clone_max_duration: 30.0  # seconds
+  voice_clone_sample_rate: 22050  # Hz
+```
+
+#### System Requirements
+- **Disk Space**: ~3GB (1.5B) or ~14GB (7B)
+- **RAM**: 8GB minimum (1.5B), 16GB (7B)
+- **VRAM**: 4GB+ (1.5B), 16GB+ (7B)
+- **Features**: 
```

markdownlint-cli2 (0.21.0)

[warning] Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)